client/src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react';
+import {
+  Container,
+  Overlay,
+  CloseOut,
+  TopBar,
+  Header,
+  TitleText,
+  ModalContentBorder,
+  ModalContentWrapper,
+  LeftSideContent,
+  TagTitle,
+  TagWrapper,
+  SideSeparator,
+  ToolTip,
+  QuestionMarkHover,
+  TagList,
+  TagsSpacer,
+  SpanTags,
+  RightSideContent,
+  SignInButton,
+  SignInHeader,
+  SignInInfo,
+} from './styles';
+import GamesContext from '../../context';
+
+function Modal() {
+  const { showModal, setModal, sidePanelInfo } = useContext(GamesContext);
+  return (
+    <>
```

Comment:
Never seen this used before - curious as to what it does as I see a closing tag ("</>") for it down below!